[SecuritySolution][Endpoint][ResponseActions] Unskip automated response action e2e test

ashokaditya opened the pull request:
## Summary


Update and unskip the test to find and kill `sshd` process instead of `agentbeat` process that is sometimes slow to start and may not be running within the test window.

### Flaky runner (ESS and serverless)
- https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/9921/steps/canvas x 50 (each)- all green

### Checklist

Check the PR satisfies following conditions. 

Reviewers should verify this PR satisfies this list as well.

- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-i18n/README.md)
- [ ] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials
- [x] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
- [ ] If a plugin configuration key changed, check if it needs to be allowlisted in the cloud and added to the [docker list](https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker)
- [ ] This was checked for breaking HTTP API changes, and any breaking changes have been approved by the breaking-change committee. The `release_note:breaking` label should be applied in these situations.
- [x] [Flaky Test Runner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was used on any tests changed
- [ ] The PR  description includes the appropriate Release Notes section, and the correct `release_note:*` label is applied per the [guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)
- [ ] Review the [backport guidelines](https://docs.google.com/document/d/1VyN5k91e5OVumlc0Gb9RPa3h1ewuPE705nRtioPiTvY/edit?usp=sharing) and apply applicable `backport:*` labels.

### Identify risks

Does this PR introduce any risks? For example, consider risks like hard to test bugs, performance regression, potential of data loss.

Describe the risk, its severity, and mitigation for each identified risk. Invite stakeholders and evaluate how to proceed before merging.

- [ ] [See some risk examples](https://github.com/elastic/kibana/blob/main/RISK_MATRIX.mdx)
- [ ] ...

ashokaditya committed:
unskip

closes https://github.com/elastic/kibana/issues/241197

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

ashokaditya committed:
update test

ashokaditya committed:
for flaky test

undo before merge

ashokaditya committed:
update test to find "sshd" process instead

find an kill sshd process instead of agentbeat that is sometimes slow to start and may not be running within the test window.

ashokaditya committed:
Merge branch 'main' into chore/olm-unskip-test-241197

kibanamachine committed:
Changes from node scripts/regenerate_moon_projects.js --update

ashokaditya committed:
for running on flaky test runner

elasticmachine commented:
## :broken_heart: Build Failed
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/364718)
* Commit: b4d5d0fc19068055ae353b7edc722e44d2258652

### Failed CI Steps
* [Linting](https://buildkite.com/elastic/kibana-pull-request/builds/364718#019ab9f3-01fa-4100-8b38-1719d5cc6a20)

### History
* :broken_heart: [Build #364450](https://buildkite.com/elastic/kibana-pull-request/builds/364450) failed e4097ea8906c31f88540629842f50922694bf847

cc @ashokaditya

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/364718","number":364718,"commit":"b4d5d0fc19068055ae353b7edc722e44d2258652"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/364450","number":364450,"commit":"e4097ea8906c31f88540629842f50922694bf847"}],"number":364718}
buildkite-pr-comment-->

ashokaditya committed:
update spec correctly for flaky test runner

kibanamachine commented:
## Flaky Test Runner Stats
### 🎉 All tests passed! - [kibana-flaky-test-suite-runner#9921](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/9921)
[✅] Security Solution Defend Workflows - Cypress: 50/50 tests passed.
[✅] [Serverless] Security Solution Defend Workflows - Cypress: 50/50 tests passed.

[see run history](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds?branch=refs%2Fpull%2F241463%2Fhead)

ashokaditya committed:
revert spec change

ashokaditya committed:
Merge branch 'main' into chore/olm-unskip-test-241197

kibanamachine committed:
Changes from node scripts/regenerate_moon_projects.js --update

elasticmachine commented:
Pinging @elastic/obs-ux-management-team (Team:obs-ux-management)

ashokaditya committed:
Merge branch 'main' into chore/olm-unskip-test-241197

shahargl reviewed (approved):
lgtm

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/365695)
* Commit: 8138e4ae7a06ae7689f1bc94f459748cc661b8a0

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :broken_heart: [Build #364731](https://buildkite.com/elastic/kibana-pull-request/builds/364731) failed e444d2913277cd9c7be768abbc1f97aa8bb14f7f
* :broken_heart: [Build #364718](https://buildkite.com/elastic/kibana-pull-request/builds/364718) failed b4d5d0fc19068055ae353b7edc722e44d2258652
* :broken_heart: [Build #364450](https://buildkite.com/elastic/kibana-pull-request/builds/364450) failed e4097ea8906c31f88540629842f50922694bf847

cc @ashokaditya

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/365695","number":365695,"commit":"8138e4ae7a06ae7689f1bc94f459748cc661b8a0"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/364731","number":364731,"commit":"e444d2913277cd9c7be768abbc1f97aa8bb14f7f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/364718","number":364718,"commit":"b4d5d0fc19068055ae353b7edc722e44d2258652"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/364450","number":364450,"commit":"e4097ea8906c31f88540629842f50922694bf847"}],"number":365695}
buildkite-pr-comment-->
